index.html

```diff
@@ -13,25 +13,31 @@
 <body>
     <header class="header-bar d-flex">
         <a class="header-title" href="#">Syed Aman</a>
-        <a class="menu-button" href="#" onclick="toggle()">
+        <nav class="menu-button"  onclick="toggle()">
             <img id="hamburger-btn" src="assets/ham.png" alt="hamburger">
-        </a>
+        </nav>
     </header>
 
     <section class="padding-section d-flex dark-section hero-section">
         <h1 class="section-title">Hey There.<br>I'm Aman</h1>
-        <p class="section-subtitle">I'm a software developer</p>
+        <h2 class="section-subtitle">I'm a software developer</h2>
         <p class="section-text hero-section-text">I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
         </p>
```

Comment:
- [ ] The former code reviewer requested that you follow the design used in Figma. Currently, you are not using the same measurements and CSS declarations used for this section of your template. It would be nice if you implemented the suggested change made by the reviewer.

## Your Design

![image](https://user-images.githubusercontent.com/77106962/144463290-c1399076-9a42-4391-9fbe-efe0061ce2cc.png)


## Figma's Design

![image](https://user-images.githubusercontent.com/77106962/144463369-1005a4a5-fd12-4c54-82a5-af49fa0b36bb.png)

#### Kindly implement the requested change 👍